sat/cli/bootsys/parser.py

```diff
@@ -139,6 +139,18 @@ def _add_bos_template_options(subparser, action):
              'xname will be expanded into the node xnames for all nodes in that slot.'
     )
 
+    subparser.add_argument(
+        '--staged-session', action='store_true',
+        help=f'If specified, then create a "staged" BOS session. A staged session differs '
+             f'from a normal BOS session in that a staged session does not automatically '
+             f'change the state of any components targeted by the session templates. '
+             f'Instead, a staged session will update components\' "staged_state", which '
+             f'can later by applied through the command \'cray bos v2 applystage create\', '
```

Comment:
nit:
```suggestion
             f'can later be applied through the command \'cray bos v2 applystage create\', '
```

Reply:
Nice catch. This typo also exists in the man page. Fixed with c814d831904beeadd4bddc5cb352d498a18874c5.